### PR TITLE
Update LTS status in README

### PR DESCRIPTION
**Pull request by bajtos**
See strongloop/loopback#4306

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-filters) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [ ] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # loopback-filters
 
+**⚠️ LoopBack 3 is in Maintenance LTS mode, only critical bugs and critical
+security fixes will be provided.**
+
+We urge all LoopBack 3 users to migrate their applications to LoopBack 4 as
+soon as possible. Refer to our
+[Migration Guide](https://loopback.io/doc/en/lb4/migration-overview.html)
+for more information on how to upgrade.
+
```
Comment (dhmlau) [inline README.md:11]: Do we wanna add the LTS schedule like https://github.com/strongloop/generator-loopback/pull/415/files? Thanks.

Comment (bajtos) [reply README.md:11]: I wouldn't bother with that. This modules is feature incomplete and out of sync with LoopBack in many edge cases. Surprisingly enough, it has over 23k downloads per month 😱 https://www.npmjs.com/package/loopback-filters

Comment (bajtos) [reply README.md:11]: Never mind, I'll add the LTS section for consistency.

Comment (bajtos) [reply README.md:11]: Done, LGTY now?

**Review by dhmlau — APPROVED** [review]

**Pull request merged**